libraries/rush-lib/src/logic/installManager/InstallHelpers.ts

```diff
@@ -48,7 +48,8 @@ export class InstallHelpers {
     };
 
     if (rushConfiguration.packageManager === 'pnpm') {
-      const { pnpmOptions } = rushConfiguration;
+      const pnpmOptions: PnpmOptionsConfiguration =
+        subspace.getPnpmOptions() || rushConfiguration.pnpmOptions;
```

Comment:
```suggestion
        subspace.getPnpmOptions() ?? rushConfiguration.pnpmOptions;
```

Reply:
Is `subspace` always defined?

Reply:
Yes, subspace gets defined as the default workspace if subspaces is not enabled, as you can see in the parameters for this function, subspace is listed as a non-optional parameter